optimizer_wrapper.rb

```diff
@@ -254,16 +254,14 @@ def self.solve(service_vrp, job = nil, block = nil)
             proc{ |pids|
               next unless job
 
-              current_result = Result.get(job) || { 'pids' => nil }
-
+              current_result = Result.get(job) || { pids: nil }
               current_result[:csv] = true if cliqued_vrp[:restitution_csv]
-
-              current_result['pids'] = pids
+              current_result[:pids] = pids
 
               Result.set(job, current_result)
             }
           ) { |wrapper, avancement, total, _message, cost, _time, solution|
-            block&.call(wrapper, avancement, total, 'run optimization, iterations', cost, (Time.now - time_start) * 1000, solution.class.name == 'Hash' && solution) if dicho_level.nil? || dicho_level.zero?
+            block&.call(wrapper, avancement, total, 'run optimization, iterations', cost, (Time.now - time_start) * 1000, solution.class.name == 'Hash' ? solution : nil) if dicho_level.nil? || dicho_level.zero?
```

Comment:
Do we need this modification? Previous code is shorter and does the same thing.

Reply:
In fact it does not do the same thing. This sometimes returned 'false' which produced an error when GET was called. 
In the case where solution.class.name == NilClass and solution is nil this returned false.